docker-compose.yml

```diff
@@ -40,7 +40,3 @@ services:
     volumes:
       # This directory must have cert files
       - ./volumes/web/cert:/cert:ro
-      - /etc/localtime:/etc/localtime:ro
```

Comment:
i added this line again, i think this change will spark a discussion that does not belong in this PR